docs/issue-labels.rst

```diff
@@ -7,93 +7,89 @@ what they stand for.
 .. _GitHub issue tracker: https://github.com/rtfd/readthedocs.org/issues
 
 *Bug*
-    An issue describing unexpected or malicious behaviour of the
-    readthedocs.org software.
+    An issue describing unexpected or malicious behaviour of the readthedocs.org
+    software. A Bug issue differs from an Improvement issue in that Bug issues
+    are given priority on our roadmap. On release, these issues generally only
+    warrant incrementing the patch level version.
 
 *Community Effort*
-    Tickets with this label are valid issues that the core team thinks are
+    Issues with this label are valid issues that the core team thinks are
     worth to fix or implement in the future. However the core team's resources
-    are too scarce to address these issues. Tickets marked with this label
+    are too scarce to address these issues. Issues marked with this label
     are issues that the core team will **not** work on, but contributions
     from the community are very welcome.
```

Comment:
I'd be tempted to just close these and remove the concept, as we've talked about in the past (#2673)

Reply:
Good point. These generally just sit around without activity. I think just closing these as non-accepted features and invalid support issues is in line with the rest of the changes here.